src/code/z_actor.c

```diff
@@ -2833,8 +2832,7 @@ Actor* Actor_Spawn(ActorContext* actorCtx, PlayState* play, s16 actorId, f32 pos
 
                 if (actorCtx->absoluteSpace == NULL) {
                     // "AMF: absolute magic field"
-                    actorCtx->absoluteSpace =
-                        ZeldaArena_MallocRDebug(ACTOROVL_ABSOLUTE_SPACE_SIZE, "AMF:絶対魔法領域", 0);
+                    actorCtx->absoluteSpace = ZELDA_ARENA_MALLOC_R(ACTOROVL_ABSOLUTE_SPACE_SIZE, "AMF:絶対魔法領域", 0);
```

Comment:
og code didn't use `__FILE__`, `__LINE__` here so it's debatable if we want to reflect that using the debug function directly or not.

For non_matching debug purposes it seems more useful to me to indeed use the macro so it would use real file and line info

this also happens for other `ZeldaArena_Malloc*` calls in this function